knora/dsplib/utils/onto_validate.py

```diff
@@ -45,7 +45,7 @@ def validate_ontology(input_file_or_json: Union[str, Dict, os.PathLike]) -> bool
     try:
         validate(instance=data_model, schema=schema)
     except jsonschema.exceptions.ValidationError as err:
-        print(err)
+        print('Data model did not pass validation. The error message is:', err.message)
         return False
```

Comment:
I think this could be simplified to just `validate(instance=data_model, schema=schema)`: Either it passes quetly, or it raises a validation error, which will cause it to exit with non-zero exit code